apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -1,3 +1,5 @@
+import React, { useCallback, useState } from "react";
```

Comment:
See how we import hooks for the rest of the application. follow the application style